examples/README.md

```diff
@@ -41,7 +41,7 @@ md_toc github examples/README.md | sed -e "s/\s-\s/ * /"
   * [hpc-slurm-chromedesktop.yaml](#hpc-slurm-chromedesktopyaml--) ![community-badge] ![experimental-badge]
   * [flux-cluster](#flux-clusteryaml--) ![community-badge] ![experimental-badge]
   * [tutorial-fluent.yaml](#tutorial-fluentyaml--) ![community-badge] ![experimental-badge]
-  * [hpc-slurm-legacy-sharedvpc.yaml](#hpc-slurm-legacy-sharedvpcyaml--) ![community-badge] ![deprecated-badge]
+  * [hpc-slurm-legacy-sharedvpc.yaml](#hpc-slurm-legacy-sharedvpcyaml--) ![community-badge]
```

Comment:
1) I bet the link is now broken. s/#hpc-slurm-legacy-sharedvpcyaml--/#hpc-slurm-legacy-sharedvpcyaml-/
2) Move up in the list above all other experimental examples

Reply:
1) I am not sure what you mean here. It should point to https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/examples#hpc-slurm-legacy-sharedvpcyaml-- one right ? I just removed the deprecated-badge. Not sure if this broke the link.

2) Done. Thanks for catching this.

Reply:
The anchor (`#hpc-slurm-legacy-sharedvpcyaml--`) is generated from the contents of L983. You have changed the contents of that line so the anchor will change.

The `-` are stand in for the badges, there used to be 2 now there is just 1, you still have 2x `-` in the anchor.

Reply:
Updated this.